builds/template-steps-build-test.yml

```diff
@@ -0,0 +1,106 @@
+parameters:
+  configuration: ''
+  solution: ''
+  nugetVersion: ''
+  runCodeSign: ''
+
+steps:
+- task: UseDotNet@2
+  displayName: 'Install .NET SDK for Build'
+  inputs:
+    version: '5.0.x'
+
+# We don't need restore now, but this may be needed in the future
+# - task: DotNetCoreCLI@2
+#   displayName: '.NET Restore'
+#   inputs:
+#     command: restore
+#     projects: '${{ parameters.solution }}'
+
+# Don't generate package so we can sign it before packaging
+- task: DotNetCoreCLI@2
+  displayName: '.NET Build'
+  inputs:
+    command: build
+    projects: '${{ parameters.solution }}'
+    arguments: '--configuration ${{ parameters.configuration }} -p:GeneratePackageOnBuild=false'
+
+# CodeSign is only run on release builds
+- task: UseDotNet@2
+  displayName: 'Install .NET Core SDK 2.1.x for CodeSign'
```

Comment:
CodeSign doesn't work with newer .NET Core SDKs?

Reply:
Looks like it's supported to .NET 5 SDK now, updated